libcontainer/cgroups/systemd/unified_hierarchy.go

```diff
@@ -117,11 +117,6 @@ func (m *UnifiedManager) Apply(pid int) error {
 			newProp("CPUQuotaPerSecUSec", cpuQuotaPerSecUSec))
 	}
 
-	if c.Resources.BlkioWeight != 0 {
-		properties = append(properties,
-			newProp("BlockIOWeight", uint64(c.Resources.BlkioWeight)))
```

Comment:
Do we have modern equivalent for this?

Reply:
Yes, it's `IOWeight` but I tested it and faced some strange error `no such device or address`, so I will continue investigation related to this error, but until that we can merge this part.

Reply:
If systemd can translate the legacy value, let's keep the legacy code for IO.